Assets/Scripts/MainMenu.cs

```diff
@@ -0,0 +1,99 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MainMenu : MonoBehaviour
+{
+    public GameObject pivotPoint; //For camera
+    public float rotationSpeed;
+    public float creditsSpeed;
+
+    bool isCreditsWindowOpen;
+
+    public Scene mainMenuScene;
+    public GameObject confirmationWindow;
+    public GameObject creditsWindow;
+    public GameObject backCreditsButton;
+    public GameObject settingsWindow;
+    public GameObject buttonsInMenu;
+    
+    Vector3 creditsStartPos = new Vector3();
+    void Start()
+    {
+        creditsStartPos = new Vector3(creditsWindow.transform.position.x, creditsWindow.transform.position.y, creditsWindow.transform.position.z);
+        buttonsInMenu.active = true;
+        settingsWindow.active = false;
```

Comment:
using `.active` is obsolete - use `.SetActive()`  instead.
Also consider moving activation & deactivation of certain objects/UI elements into function. Like `ActiveSettingsMenu()` etc.